lib/app.js

```diff
@@ -8,6 +8,7 @@ var mbaasApi = require('fh-mbaas-api')
   , cors = require('cors')
   , mediator = require('fh-wfm-mediator/lib/mediator')
   , bodyParser = require('body-parser')
+  , Promise = require('bluebird')
```

Comment:
Style: can you take the opportunity to change this commas to separate `var` statements? :D